xbox/xbox.c

```diff
@@ -1155,6 +1265,9 @@ GL_API void GL_APIENTRY glBufferSubData (GLenum target, GLintptr offset, GLsizei
   assert(buffer->size >= (offset + size));
   assert(data != NULL);
   memcpy(&buffer->data[offset], data, size);
+debugPrint("Set %d bytes at %d in %p\n", size, offset, &buffer->data[offset]);
+  memset(&buffer->data[offset], 0xFF, size);
+Sleep(100);
```

Comment:
Somehow this is not setting the data properly? That 0xFF pattern is never even visible.
It seems to rarely [never?] set offset 0

Reply:
I don't remember if I ever solved this? I assume it works, at least for some buffers